README.md

```diff
@@ -510,6 +511,20 @@ You can set `vscode.notify` as your default notify functions.
 vim.notify = vscode.notify
 ```
 
+##### `vscode.get_status_item(id)`
+
+Creates a status item
```

Comment:
I have no idea what this does. Why is it needed? Why can't users use `:echo` or something like that?